### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by yarodevuci** [StateOfCalifornia/DigitalCovid19VaccineRecord-QR#2]
How to use this? VS code does not recognized start file...

**Comment by michaeljivan7** [comment]
Hi @yarodevuci,

Please clone the GitHub repository via command line
`git clone https://github.com/StateOfCalifornia/DigitalVaccineRecord-QR.git`

Using Visual Studio Code add the Azure Functions extension, from the extensions installer.

Ensure this step by install azure functions core tools`npm install -g azure-functions-core-tools@3`

Now Run > Start Debugging

Make note of the local URL for API secrets. 
`Example: http://localhost:7071/api/QRCreate`

This value will be used in the API, under the secrets.json file property { "AppSettings:QrCodeApi": "`insert url here`"}

#Functions QRCreate: Generates a QR code of the shc string passed from the API.

**Comment by yarodevuci** [comment]
Hi @michaeljivan7 
Thanks, will give it a try

**Comment by montyJordanIII** [comment]
does this have to be run using VSC debugger?  I noticed npm start is mapped to func start and that there are dependencies, shouldn't npm install be run?

**Comment by montyJordanIII** [comment]
> Hi @yarodevuci,
> 
> Please clone the GitHub repository via command line `git clone https://github.com/StateOfCalifornia/DigitalVaccineRecord-QR.git`
> 
> Using Visual Studio Code add the Azure Functions extension, from the extensions installer.
> 
> Ensure this step by install azure functions core tools`npm install -g azure-functions-core-tools@3`
> 
> Now Run > Start Debugging
> 
> Make note of the local URL for API secrets. `Example: http://localhost:7071/api/QRCreate`
> 
> This value will be used in the API, under the secrets.json file property { "AppSettings:QrCodeApi": "`insert url here`"}
> 
> #Functions QRCreate: Generates a QR code of the shc string passed from the API.

The extensions manager does not install core tools and fails right away with a message about installing core tools as a dependency.

**Comment by michaeljivan7** [comment]
Hi @montyJordanIII ,

Azure functions core is required to run the function app correctly.
Were you able to successfully add the Azure functions extension to Visual Studio Code?

**Comment by montyJordanIII** [comment]
> Hi @montyJordanIII ,
> 
> Azure functions core is required to run the function app correctly. Were you able to successfully add the Azure functions extension to Visual Studio Code?

Yes, I was able to add the azure core and function extensions into VSC.  Not quite sure yet if the app is functioning properly yet.  Is there a way to test it out without the other UI & API components?

**Comment by michaeljivan7** [comment]
> > Hi @montyJordanIII ,
> > Azure functions core is required to run the function app correctly. Were you able to successfully add the Azure functions extension to Visual Studio Code?
> 
> Yes, I was able to add the azure core and function extensions into VSC. Not quite sure yet if the app is functioning properly yet. Is there a way to test it out without the other UI & API components?

You can try calling your local function app with payload of an shcs code,

Example:
Create a POST call to http://localhost:0000/api/QRCreate and add to the body a payload of an shc string "shc:/56762909524"
